controllers/storagecluster/provider_server.go

```diff
@@ -26,8 +25,7 @@ const (
 	ocsProviderServerName  = "ocs-provider-server"
 	providerAPIServerImage = "PROVIDER_API_SERVER_IMAGE"
 
-	ocsProviderServicePort     = int32(50051)
-	ocsProviderServiceNodePort = int32(31659)
+	ocsProviderServicePort = int32(50051)
```

Comment:
Will it be possible to make the port configurable i.e part of the CRD?

Reply:
If I understand correctly @oritwas is asking Can this port be configurable like the user can provide a custom port when deploying OCS in provider mode? like in CSV/CRD or POD ENV etc?

Reply:
not sure POD ENV is useful

Reply:
We can read it from the storagecluster CR if required. I am not in favor of the ENV variables as we have to populate that value either via CSV on the build time or via configmap.

Reply:
Why complicate things? just have it as a constant. This is for provider mode which is always deployed on its own cluster which means that the chance of collision is almost 0. So why are trying to solve a non-issue